src/screens/Lending/ViewDepositedAsset.js

```diff
@@ -230,7 +230,7 @@ const ViewDepositedAsset = ({
                 }),
               })}
             </MediumText>
-            <BaseText success fontSize={fontSizes.medium}>
```

Comment:
If `success` is no longer present I wonder if it's debt here of change to `positive` 🤔

Reply:
fixed